tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -1296,8 +1341,7 @@ public function theUserCreatesAFolderToAnotherOwnerSpaceUsingTheGraphApi(
 		string $spaceName,
 		string $ownerUser = ''
 	): void {
-		$response = $this->createFolderInSpace($user, $folder, $spaceName, $ownerUser);
-		$this->featureContext->setResponse($response);
+		$this->createFolderInSpaceAndSetResponse($user, $folder, $spaceName, $ownerUser);
```

Comment:
previous one is the right way to set reponse
```suggestion
		$response = $this->createFolderInSpace($user, $folder, $spaceName, $ownerUser);
		$this->featureContext->setResponse($response);
```